homeassistant/components/ezviz/config_flow.py

```diff
@@ -384,6 +385,7 @@ async def async_step_reauth_confirm(
         return self.async_show_form(
             step_id="reauth_confirm",
             data_schema=data_schema,
+            description_placeholders={CONF_NAME: self._get_reauth_entry().title},
```

Comment:
We should not be handling it this way. Let's see if we can make it smarter and inject name automatically. This is not something every integration should have to do.

Reply:
I have made a proposal in https://github.com/home-assistant/core/pull/129232 to solve it so it's automatically added when missing.

Reply:
Shall we keep this open as backup or can I close the PR?